packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -1236,6 +1236,9 @@ export class DeltaManager
             clientId: connection.clientId,
             mode: connection.mode,
         };
+        if (connection.relayServiceAgent !== undefined) {
+            this.connectionStateProps.relayServiceAgent = connection.relayServiceAgent;
```

Comment:
Is there any way to get this property to _Connected event in container? I believe this.connectionStateProps we use in very few places, mostly to debug connectivity issues. Take a look at DeltaManager.connectionProps

Reply:
Added there.